src/components/Inngest.ts

```diff
@@ -339,11 +342,22 @@ export class Inngest<
       headers: { ...this.headers },
     });
 
-    if (response.status >= 200 && response.status < 300) {
-      return;
-    }
+    let rawBody: unknown;
+
+    try {
+      rawBody = await response.json();
 
-    throw await this.#getResponseError(response);
+      const body = await z
+        .object({
+          ids: z.array(z.string()),
+          status: z.number().min(200).max(299),
+        })
+        .parseAsync(rawBody);
```

Comment:
If we decide at some point to return more data from the response and the shape of the response changes, would this parsing fail? Or will it only parse the data expected and ignore the rest?